src/modules/RootPart.rb

```diff
@@ -1682,14 +1682,16 @@ def MountPartitions(root_device_current)
           Yast2::FsSnapshotStore.save("update", snapshot.number)
         end
         Update.clean_backup
+        pre_inject_files
         create_backup
         inject_intsys_files
       end
 
       success
     end
 
-    RESOLV_CONF = "/etc/resolv.conf".freeze
+    RESOLV_CONF = "/run/netconfig/resolv.conf".freeze
```

Comment:
Hm, writing to `/run` might make troubles. In Leap 15.0 the `/run` is on a tmpfs file system (RAM disk), but there is no `/etc/fstab` entry for it so during upgrade `/mnt/run` is actually on the disk.

There is some extra cleanup done in the end (https://github.com/yast/yast-installation/pull/725), but I think this should be mentioned in a comment as this behavior is not obvious from the code.